### PR TITLE
docs: switch and text

### LINKED ISSUE #580: pydocs for switch

### LINKED ISSUE #581: pydocs for text

### PR DESCRIPTION
**Pull request by ethanalvizo**
Closes #580 
Closes #581

### CODE REVIEW

File: plugins/ui/src/deephaven/ui/components/__init__.py [hunk 1]
```diff
@@ -3,8 +3,6 @@
 from .action_menu import action_menu
 from .basic import (
     component_element,
```
Comment (mofojed) [inline plugins/ui/src/deephaven/ui/components/__init__.py:5]: We could just move this to it's own file `component_element.py` now since it's the only thing in here. But do it as a separate PR, as we'll need to change a bunch of imports

**Review by mofojed — APPROVED** [review]

File: plugins/ui/src/deephaven/ui/components/__init__.py (hunk 1 above)
Comment (ethanalvizo) [reply plugins/ui/src/deephaven/ui/components/__init__.py:5]: Filed into own ticket #646
